pkg/config/constant/api.go

```diff
@@ -103,6 +103,9 @@ const (
 	// DefaultRequiredIgws number of internet gateways required for cluster creation
 	DefaultRequiredIgws = 1
 
+	// DefaultRequiredFargateOnDemandPods is the minimum number of fargate pods that should be available
+	DefaultRequiredFargateOnDemandPods = 50
```

Comment:
Why 50? Just a seemingly high enough number?

Reply:
We can easily use 20->30 pods before we add custom applications, 50 is fairly conservative.

Reply:
How do we(or the user) increase this if the user hits 50?

Reply:
Perhaps the name is a little misleading, read this as a minimum of 50 pods should be available, the AWS default is 1000 fargate ondemand pods (calculated across ecs and eks)

A user of the AWS account would have to request a service quota increase